### PR TITLE
Prepare for rel-1.17.0

### PR DESCRIPTION
**Pull request by roborags**
### Description
Prepare for rel-1.17.0 branch cut (planned for 08/06 [https://github.com/onnx/onnx/wiki/Logistics-for-ONNX-Release-1.17.0](https://github.com/onnx/onnx/wiki/Logistics-for-ONNX-Release-1.17.0))

### Motivation and Context
ONNX 1.17.0 is planned to be released on 08/20

### CODE REVIEW

File: onnx/test/helper_test.py [hunk 1]
```diff
@@ -410,12 +410,14 @@ def test(opset_versions: list[tuple[str, int]], ir_version: int) -> None:
...
         test([("", 20)], 9)
         test([("", 21)], 10)
+        test([("", 22)], 10)
         # standard opset can be referred to using empty-string or "ai.onnx"
         test([("ai.onnx", 9)], 4)
         test([("ai.onnx.ml", 2)], 6)
         test([("ai.onnx.ml", 3)], 8)
         test([("ai.onnx.ml", 4)], 9)
         test([("ai.onnx.ml", 5)], 10)
+        test([("ai.onnx.ml", 6)], 10)
```
Comment (cjvolzka) [inline onnx/test/helper_test.py:420]: Is there a new ai.onnx.ml opset? I haven't been following the 1.17 changes so I'm not sure. 

The info I can gleen from the code I see seems inconsistent:
* https://onnx.ai/onnx/operators/index.html (which I believe is based on `main`) on the `ai.onnx.ml` tab, I only see up to `5`.
* https://github.com/onnx/onnx/blob/main/onnx/helper.py#L80 does show a `6`

You might need to reach out to the Operators team to get clarity and either get the documentation or helper.py updated.

**Review by cjvolzka — COMMENTED** [review]
Based on the [PR for 1.16.0](https://github.com/onnx/onnx/pull/5959/files), it looks like a couple more places also need to be updated:
* https://github.com/onnx/onnx/blob/main/onnx/test/helper_test.py#L412 only has up to Opset `21`, meaning `22` needs to be added.
* https://github.com/onnx/onnx/blob/main/onnx/test/helper_test.py#L418 only has up to `5` meaning `6` needs to be added (if there is a ai.onnx.ml Opset 6, see in code comment)

**Comment by gramalingam** [comment]
@liqunfu : can you take a look at this PR please, thanks!

File: onnx/test/helper_test.py (hunk 1 above)
Comment (gramalingam) [reply onnx/test/helper_test.py:420]: No, I don't think there have been any updates to ai.onnx.ml opset ... eg., [this file](https://github.com/onnx/onnx/blob/99e1fd352c05c3176770080824fd7a8c474c97c0/onnx/defs/operator_sets_ml.h#L105)  is the source for onnxml opsets

Comment (roborags) [reply onnx/test/helper_test.py:420]: I reviewed all the commits from previous release upto now, there is no reference to update to ai.onnx.ml. I had updated the above helper_test.py as there was an update to [helper.py](https://github.com/onnx/onnx/blob/main/onnx/helper.py#L80) in this PR : [6099](https://github.com/onnx/onnx/pull/6099)

I will follow up [thiagocrepaldi](https://github.com/thiagocrepaldi) to check if this was expected.

Comment (roborags) [reply onnx/test/helper_test.py]: Confirmed with [thiagocrepaldi](https://github.com/thiagocrepaldi) that the opset number in this file can be rolled back.

**Comment by roborags** [comment]
@liqunfu  @gramalingam  Can you kindly review and let me know if additional changes are needed.

**Review by liqunfu — APPROVED** [review]

**Review by justinchuby — APPROVED** [review]

File: onnx/test/helper_test.py (hunk 1 above)
Comment (rnareshs) [reply onnx/test/helper_test.py]: Resolving comment as changes are reverted

**Comment by justinchuby** [comment]
Ready to merge?

**Comment by roborags** [comment]
Merging to main.

**Pull request merged**